### DISCUSSION TITLE
Publicitario para Front-End

### DISCUSSION THREAD

**Discussion by rodie77** [frontendbr/forum#1378] [Discussão]
Sou estudante de Publicidade e Propaganda mas por certa insatisfação com área pretendo migrar para TI. Dito isso, gostaria de fazer algumas perguntas:

1) É possível ir para TI(Ux Design/FrontEnd) sem necessariamente ter que mudar de curso?
2) Certificados serão fundamentais?
3) Cursos do udemy e do origamid são bons?

**Comment by nandomoreirame** [comment]
Segue meus 10 centavos:

1. É possível sim, eu mesmo ainda não tenho formação, comecei 3 faculdades e tranquei, trabalho na área há quase 8 anos e nem sei se farei faculdade na área, estou pretendendo fazer alguma facul bem foda da área de TI;
2. Certificados raramente exigidos pra trabalhar na área, algumas empresas grandes já estão deixando de exigir o canudo do desenvolvedor na hora de contratar;
3. Todo e qualquer curso é válido para aumentar sua capacidade de formação intelectual, não só de TI mas de outras áreas também são válidos. Eu fiz e façao alguns cursos na Udemy, mas desse Origamid nunca fiz.

Valeu e boa sorte nessa nova empreitada! 🤘

**Comment by tunnes** [comment]
Eu acho bem tranquilo você prosperar como desenvolvedor mesmo sem uma graduação relacionada, sobre as questões que apontou:

- Ta no começo do curso e de qualquer maneira você vai continuar uma gradução? Eu optaria por mudar, pois mesmo que muitas vezes desatualizada e chata a grade curricular, você terá mais contato com lógica de programação e outros topicos chave que podem te ajudar a pegar coisas atuais do mercado com mais facilidade ou vão te dar autonomia em N situações.

- Certificado não é algo necessário, muito provavelmente nunca vão te pedir, é bacana você focar em ter dominio okay em algum tópico, mas todo destaque é valido (até mesmo um curso de verão).
 
-  Sim são bons sim, ainda mais com treino e testando se você realmente pegou o que a aula aborda.

Boa sorte!

**Comment by rodie77** [comment]
> Segue meus 10 centavos:
> 
> 1. É possível sim, eu mesmo ainda não tenho formação, comecei 3 faculdades e tranquei, trabalho na área há quase 8 anos e nem sei se farei faculdade na área, estou pretendendo fazer alguma facul bem foda da área de TI;
> 2. Certificados raramente exigidos pra trabalhar na área, algumas empresas grandes já estão deixando de exigir o canudo do desenvolvedor na hora de contratar;
> 3. Todo e qualquer curso é válido para aumentar sua capacidade de formação intelectual, não só de TI mas de outras áreas também são válidos. Eu fiz e façao alguns cursos na Udemy, mas desse Origamid nunca fiz.
> 
> Valeu e boa sorte nessa nova empreitada! 🤘

Muito obrigado pela ajuda, amigo. Já estou começando alguns cursos, agora resta ver se tenho talento pra isso. hahaha

> Eu acho bem tranquilo você prosperar como desenvolvedor mesmo sem uma graduação relacionada, sobre as questões que apontou:
> 
> * Ta no começo do curso e de qualquer maneira você vai continuar uma gradução? Eu optaria por mudar, pois mesmo que muitas vezes desatualizada e chata a grade curricular, você terá mais contato com lógica de programação e outros topicos chave que podem te ajudar a pegar coisas atuais do mercado com mais facilidade ou vão te dar autonomia em N situações.
> * Certificado não é algo necessário, muito provavelmente nunca vão te pedir, é bacana você focar em ter dominio okay em algum tópico, mas todo destaque é valido (até mesmo um curso de verão).
> * Sim são bons sim, ainda mais com treino e testando se você realmente pegou o que a aula aborda.
> 
> Boa sorte!

Falta apenas 1 ano e meio para eu concluir, então não sei se mudar de curso seria uma boa. Mas pretendo fazer uma pós em UX ou um tecnólogo em algo de TI.

Aliás, o mercado é tão bom assim mesmo? Como uma pessoa de fora, um profissional com um conhecimento em HTML, CSS, JavaScript já tem uma boa perspectiva de mercado.

**Comment by vitocaio** [comment]
Cara no geral o que tenho visto no mercado de TI é a comprovação pratica dos seus conhecimentos, através  de code challangers, se tu passar não vão te pedir nem RG ( brincadeira rs ).

**Comment by vicainelli** [comment]
Fala @rodie77 , respondendo suas perguntas.

1. Sim, totalmente possível, trabalho com desenvolvimento (front/back) desde 2009, sou formado em publicidade e propaganda, comecei a me interessar por desenvolvimento web nas aulas de html e css do curso, fiz um curso básico de HML/CSS/JS no Senac, na época foi bom, além de pegar livros na biblioteca na faculdade pra tentar entender melhor o que era esse mundo.
2. Não.
3. Tem muito curso bom, assim como tem outros que não são, em geral, olho a grade, se ele é atualizado com frequência, e também comparo os reviews. (só uso Udemy)

**Comment by rodie77** [comment]
> Cara no geral o que tenho visto no mercado de TI é a comprovação pratica dos seus conhecimentos, através de code challangers, se tu passar não vão te pedir nem RG ( brincadeira rs ).

Bom saber isso, to cansado de dinâmicas de grupo com 30 pessoas que é a solução de um case em conjunto. hahaha



> Fala @rodie77 , respondendo suas perguntas.
> 
> 1. Sim, totalmente possível, trabalho com desenvolvimento (front/back) desde 2009, sou formado em publicidade e propaganda, comecei a me interessar por desenvolvimento web nas aulas de html e css do curso, fiz um curso básico de HML/CSS/JS no Senac, na época foi bom, além de pegar livros na biblioteca na faculdade pra tentar entender melhor o que era esse mundo.
> 2. Não.
> 3. Tem muito curso bom, assim como tem outros que não são, em geral, olho a grade, se ele é atualizado com frequência, e também comparo os reviews. (só uso Udemy)

Desculpe perguntar isso mas você trabalho hoje na área de Publicidade?

De resto, muito obrigado pela ajuda, Comecei um curso de pacote Adobe no Udemy e não gostei tanto, vou dar uma nova chance pra algum de UX.

**Comment by vicainelli** [comment]
@rodie77 eu comecei trabalhando numa agência, depois passei por 2 empresas trabalhando com produto, hoje, voltei pra agência.
Eu curto o mercado publicitário, não me arrependo de ter feito PP, me ajudou muito com uma visão de marketing para alguns freelas e projetos pessoais.
Eu só teria me empenhado mais em aprender lógica de programação, eu não dei muito bola pra isso, na época o jQuery bombava, e tudo era facilmente resolvido com 3 ou 4 links do stackoverflow / forum imasters.
Fora isso, a Publicidade me ajudou muito.

**Comment by rodie77** [comment]
> @rodie77 eu comecei trabalhando numa agência, depois passei por 2 empresas trabalhando com produto, hoje, voltei pra agência.
> Eu curto o mercado publicitário, não me arrependo de ter feito PP, me ajudou muito com uma visão de marketing para alguns freelas e projetos pessoais.
> Eu só teria me empenhado mais em aprender lógica de programação, eu não dei muito bola pra isso, na época o jQuery bombava, e tudo era facilmente resolvido com 3 ou 4 links do stackoverflow / forum imasters.
> Fora isso, a Publicidade me ajudou muito.

 Confesso que meu sonho é trabalhar em agência, espero conseguir estágio em alguma até o fim do curso. Processo seletivo mais longe que cheguei foi logo na Ogilvy. hahahaha

Eu quero focar em programação até para já ser um plano B caso meu ingresso no mercado publicitário não dê muito certo, as exigências tem sido cada vez maiores para um simples estágio de Design. Exige-se HTML, CSS, JavaScript e etc. Até por conta disso que veio meu interesse.

Bom, ajudou muito, só tenho a agradecer, amigo!!

**Comment by israelcena** [comment]
Amigo, 

Sou Formado em Marketing e também estou aprendendo programação (FrontEnd). 
Não acho que são funções distintas, muito pelo contrário, considero um complemento para desenvolver melhores aplicações, focadas na satisfação do usuário (cliente), e por exemplo, usuando a análise Swot para identificar melhorias na minha API. 
Muito melhor do que ser apenas um programador (alguém que junta códigos).

Cada qualificação complementa e não divide !

**Comment by didiraja** [comment]
Me formei em design e virei Front-End, plenamente possível. Até porque tem um monte de vaga em agência de publicidade.

**Comment by easbarba** [comment]
1. Para se tornar Front-end nao eh necessario curso. 
Mas muitas vagas exigem cursos TI como preferencia. 
OU voce ter um portifolio forte no Github.

2. Certificados sao bons se sua area tem muitas empresas publicas ou grandes empresas como Google..

3. Udemy eh otimo para introducao, nada alem. 
Se quiser estudar bem use DevMedia/Alura/Caelum/FreeCodeCamp e outros cursos completos.
Ou livros (Learning Web Design / Eloquent Javascript.

**Comment by ghost** [comment]
Meu amigo, nessa vida tudo é possível! Fiz 30 agora e, depois de uns 10 anos como jornalista, estou investindo forte em aprender JavaScript direito. Como está nessa jornada? Vamos trocar umas experiências! Tô seguindo no FreeCodeCamp e também usando os cursos do Colt Steele na Udemy.

**Comment by lysaabrantes** [comment]
Eu estou literalmente na mesma! Estou no último mês de Publicidade e Propaganda, já estou formada. Porém, sinto a necessidade de migrar de área, pois atualmente moro na Europa e sinto que somente nessa área terei algum reconhecimento.. e sempre fui uma pessoa que gostou de formatar uns computadores, configurar alguns roteadores hahaha enfim... e tb curto mto a parte de design. Mas é isso, não sei por onde começar nesse momento, só sinto que preciso fazer algo que me alavanque pra essa área da tecnologia.
